### ISSUE TITLE
Add a badge on layer button to show number of active layers

### ISSUE THREAD

**Issue by sagarpreet-chadha** [publiclab/leaflet-environmental-layers#201]
The title is self explainatory . Adding prototype soon here :smile: .

**Comment by crisner** [comment]
I would like to give this one a try. Can I work on this @sagarpreet-chadha ? May I know what kind of style you have in mind for the badge? Is it supposed to be displayed next to the 'Turn on layers' text on the leaflet control?

**Comment by sagarpreet-chadha** [comment]
Yes please. 
Have you seen whatsapp app --- the small badge that appears on contact tab that displays number of unread chats.

**Comment by crisner** [comment]
Yes. A green circle displaying the number. What color do we use here for the circle?

**Comment by crisner** [comment]
How about something between grey(leaf controls border color) and the text color with the numbers in white?

**Comment by sagarpreet-chadha** [comment]
I guess you can start writing the CSS and JS to change number in the circle .
Later we can ask @ebarry to chime in and tell the color combination . Thanks!

**Comment by crisner** [comment]
Okay 👍
